client/stylesheets/_toCat.scss

```diff
@@ -2980,26 +2982,30 @@ i:hover {
 
 .slider label {
     /*Lets add some spacing for the thumbnails*/
-    margin: 9px;
-    border: 2px solid $color-spaceblue;
-    border-radius: 5px;
+    margin: 0;
+    border: 6px solid #f8504b;
+    border-top: 3px solid #f8504b;
     float: left;
     cursor: pointer;
 }
 
+.sliderSmallTwo, .sliderSmallThree, .sliderSmallFour {
```

Comment:
Selector `sliderSmallTwo` should be written in lowercase with hyphens<br>Selector `sliderSmallThree` should be written in lowercase with hyphens<br>Selector `sliderSmallFour` should be written in lowercase with hyphens<br>Each selector in a comma sequence should be on its own line